definitions/procedures/prep_6_10_upgrade.rb

```diff
@@ -0,0 +1,16 @@
+class Procedures::Prep610Upgrade < ForemanMaintain::Procedure
+  metadata do
+    description 'Prepare for the Satellite 6.10 upgrade'
+
+    confine do
+      feature(:satellite) && feature(:satellite).current_minor_version == '6.9'
+    end
+  end
+
+  def run
+    FileUtils.chmod_R "g=rwX", "/var/lib/pulp/content"
+    puts execute!('find /var/lib/pulp/content -type d -perm -g-s -exec sudo chmod g+s {} \;')
+    FileUtils.chown_R nil, "pulp", "/var/lib/pulp/content"
+    # TODO: Install Pulp 3 without starting services?
```

Comment:
It's not certain yet if the Pulp 3 foreman-installer command should be run here.  It can always be added later if we need it.

Reply:
@ianballou instead of puts can you use spinner ? 
As it maybe time consuming can we put a nice banner, something like, `This may take long time to complete depending on the size of pulp content`. ?
I think we dont require sudo in find command because by default foreman-maintain is expected to run as root user ?

Reply:
With current confine block the procedure will be available as soon as user upgrades to 6.9 version, should we make it available only when system is upgradable to 6.10 ?